### ISSUE TITLE
Module Admin - Duplication of database numbers

### ISSUE THREAD

**Issue by rompik** [eryar/PipeCAD#17]

**Comment by rompik** [comment]
System didn't check if new db will be created with existing database number. Next code was used for massive database creation:
```python 
PipeCad.CommitTransaction()
PipeCad.StartTransaction("Create Database")
PipeCad.SetCurrentItem("/*CA")
Dbli = PipeCad.CurrentItem().Member[0]
PipeCad.SetCurrentItem(Dbli)
PipeCad.CreateItem("DB", "*CA/DESI-X-XXX-001")
aDbItem = PipeCad.CurrentItem()
aDbItem.Description = "ARCHITECTURAL"
aDbItem.DbType = "DESI"
aDbItem.DbNumber = 2000
aDbItem.CreateDatabase("X-XXX-001/CA")
PipeCad.CommitTransaction()
PipeCad.SaveWork()
```

![Duplication of db numbers](https://user-images.githubusercontent.com/12736532/165119166-0d58f8c3-e6c2-42d9-859d-1b2586424d87.png)

**Comment by rompik** [comment]
Closed by mistake

**Comment by eryar** [comment]
Fixed PipeCAD-1.0.9
